client/lcd/lcd_test.go

```diff
@@ -872,23 +872,23 @@ func getBondingTxs(t *testing.T, port string, delegatorAddr sdk.AccAddress, quer
 	return txs
 }
 
-func getDelegatorValidators(t *testing.T, port string, delegatorAddr sdk.AccAddress) []stake.BechValidator {
```

Comment:
How come no more `BechValidator`?

Reply:
the validator Keeper only returns `Validator`. Do you like me to return bech32 addresses instead ?

Reply:
`sdk.ValAddress` and `sdk.AccAddress` automatically marshal to Bech32 when using `.String()` or `.MarshalJSON()`

Reply:
Wait, `validator.BechValidator()` is returning the `ValAddress` in hex for the operator and I'm pretty sure the prefix for the pubkey is also wrong. It's using `cosmosconspub` were it should be `cosmosvalpub`. cc: @cosmos/cosmossdk